lib/util/merge-confidence/index.ts

```diff
@@ -25,14 +25,51 @@ export const confidenceLevels: Record<MergeConfidence, number> = {
 export function initConfig(): void {
   apiBaseUrl = getApiBaseUrl();
   token = getApiToken();
+  supportedDatasources = parseSupportedDatasourceString() ?? [
```

Comment:
do we really need to clone the array? do we change it later? if not just assign it ( small perf impact)